src/schema/nodes/affiliations.ts

```diff
@@ -20,7 +20,7 @@ import { NodeSpec } from 'prosemirror-model'
 // It exists for the purpose of styling in the UI
 
 export const affiliations: NodeSpec = {
-  content: 'affiliation*',
+  content: 'affiliation* paragraph*',
```

Comment:
this change should be only a change in a view of the node, why do we need to change the schema?

Reply:
this change is reverted....